drivers/display/ssd_1306/ssd_1306.c

```diff
@@ -57,6 +58,25 @@ const struct display_controller_ops ssd1306_ops = {
 
 extern const uint8_t no_os_chr_8x8[128][8];
 
+int32_t ssd1306_buffer_transmit(ssd_1306_extra *extra_param, uint8_t *command,
+				int length)
+{
+	int32_t ret;
+
+	if (extra_param->comm_type == SSD1306_SPI) {
+		ret = no_os_spi_write_and_read(extra_param->spi_desc, command, length);
+	} else if (extra_param->comm_type == SSD1306_I2C) {
+		uint8_t buff_tmp[length + 1];
+		buff_tmp[0] = 0x00;
+		memcpy(&buff_tmp[1], command, length);
+		ret = no_os_i2c_write(extra_param->i2c_desc, buff_tmp, length + 1, 1);
+	} else {
```

Comment:
hmm, maybe this else can be dropped if ret is initialized from the start with an error code.